<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="sachin-PC" ignored="0" name="RetryMultipleFailedTest" tests="16" failures="4" timestamp="27 Feb 2021 14:57:35 GMT" time="201.632" errors="0">
  <testcase name="test1" time="0.01" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.002" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.004" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.002" classname="MyTests.AnyTestPage">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at MyTests.AnyTestPage.test1(AnyTestPage.java:32)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.001" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.002" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.003" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.001" classname="MyTests.AnyTestPage">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at MyTests.AnyTestPage.test2(AnyTestPage.java:40)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test3" time="0.002" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.002" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.001" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.001" classname="MyTests.AnyTestPage">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at MyTests.AnyTestPage.test3(AnyTestPage.java:46)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test4" time="0.003" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test4 -->
  <testcase name="test4" time="0.002" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test4 -->
  <testcase name="test4" time="0.001" classname="MyTests.AnyTestPage">
    <skipped/>
  </testcase> <!-- test4 -->
  <testcase name="test4" time="0.001" classname="MyTests.AnyTestPage">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at MyTests.AnyTestPage.test4(AnyTestPage.java:51)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
</testsuite> <!-- RetryMultipleFailedTest -->
